drivers/eaton-pdu-marlin-mib.c

```diff
@@ -709,7 +722,11 @@ static snmp_info_t eaton_marlin_mib[] = {
 	/* outletControlSwitchable */
 	{ "outlet.%i.switchable", ST_FLAG_STRING, SU_INFOSIZE,
 		".1.3.6.1.4.1.534.6.6.7.6.6.1.9.%i.%i",
-		"no", SU_OUTLET | SU_TYPE_DAISY_1, &marlin_outlet_switchability_info[0] },
+		"no", SU_OUTLET | SU_FLAG_UNIQUE | SU_TYPE_DAISY_1, &marlin_outlet_switchability_info[0] },
+	/* FIXME: handle non switchable units (only measurements), which do not expose this OID */
```

Comment:
note that this FIXME could have been removed, since the absence of the pointed OID (on non-switchable units) will result in the non publication of the data